# Copyright (c) 2024 Smart Rollerz e.V. All rights reserved.

object_detection_node:
  ros__parameters:
    debug: False
    image_topic: "/camera/image/undistorted"
    object_topic: "/object_detection/object"
    sign_topic: "/object_detection/sign"
    debug_image_topic: "/object_detection/debug/image"
    state_machine_topic: "/state_machine/debug/state"
    active: True
    dt: 0.0002
    config_path: "config/model.yaml"
